android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Nov 13 23:45:10 CST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```

Comment:
Assuming this is ok going down in version?

Reply:
I noticed that. I actually don't know lol. It's autogenerated by the capacitor add android pipeline. I think it's fine though - if there were an issue it would break the app build I think since gradle is the build tool